### PR TITLE
groups_resources

### PR DESCRIPTION
**Pull request by N-Patarov**

### CODE REVIEW

File: lib/studygroup_web/router.ex [hunk 1]
```diff
@@ -3,6 +3,8 @@ defmodule StudygroupWeb.Router do
 
   import StudygroupWeb.UserAuth
 
+  resources "/groups", GroupsController
```
Comment (rmilushev) [inline lib/studygroup_web/router.ex]: Please, remove it from here.

File: lib/studygroup_web/router.ex [hunk 2]
```diff
@@ -66,6 +68,7 @@ defmodule StudygroupWeb.Router do
     put "/users/settings/update_password", UserSettingsController, :update_password
     put "/users/settings/update_email", UserSettingsController, :update_email
     get "/users/settings/confirm_email/:token", UserSettingsController, :confirm_email
+    put "/groups", GroupsController, :create
```
Comment (rmilushev) [inline lib/studygroup_web/router.ex]: Please, change to :
```
resources "/groups", GroupsController
```

**Review by rmilushev — CHANGES_REQUESTED** [review]

File: lib/studygroup/groups/group.ex [hunk 3]
```diff
@@ -0,0 +1,18 @@
+defmodule Studygroup.Groups.Group do
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  schema "groups" do
+    field :age, :integer
+    field :name, :string
+
```
Comment (ykostov) [inline lib/studygroup/groups/group.ex:8]: gotta add email field with value "string"

File: lib/studygroup_web/router.ex [hunk 4]
```diff
@@ -3,7 +3,7 @@ defmodule StudygroupWeb.Router do
 
   import StudygroupWeb.UserAuth
 
-  resources "/groups", GroupsController
+  resources "/groups", GroupController
```
Comment (ykostov) [inline lib/studygroup_web/router.ex]: Please, remove this one from this - you don't need it

File: lib/studygroup_web/templates/layout/_user_menu.html.eex [hunk 5]
```diff
@@ -2,6 +2,6 @@
   <%= link "Settings", to: Routes.user_settings_path(@conn, :edit), class: "btn btn-sm btn-link" %>
   <%= link "Logout", to: Routes.user_session_path(@conn, :delete), method: :delete, class: "btn btn-sm btn-link" %>
 <% else %>
-  <%= link "Sign In", to: Routes.user_session_path(@conn, :new), class: "btn btn-sm btn-link" %>
-  <%= link "Get Started", to: Routes.user_registration_path(@conn, :new), class: "btn btn-sm btn-primary" %>
+  <!--<%= link "Sign In", to: Routes.user_session_path(@conn, :new), class: "btn btn-sm btn-link" %>-->
+  <%= link "Sign in", to: Routes.user_session_path(@conn, :new), class: "btn btn-sm btn-primary" %>
```
Comment (ykostov) [inline lib/studygroup_web/templates/layout/_user_menu.html.eex:6]: Good job on pairing the links

File: lib/studygroup_web/templates/layout/root.html.leex [hunk 6]
```diff
@@ -21,8 +21,10 @@
             <%= link "Study books", to: Routes.page_path(@conn, :index), class: "btn btn-sm btn-link" %>
             <%= link "Study screencasts", to: Routes.page_path(@conn, :index), class: "btn btn-sm btn-link" %>
             <%= link "Study blogs", to: Routes.page_path(@conn, :index), class: "btn btn-sm btn-link" %>
+            <%= link "Groups", to: Routes.page_path(@conn, :index), class: "btn btn-sm btn-link" %>
+            <%= link "Students", to: Routes.page_path(@conn, :index), class: "btn btn-sm btn-link" %>
             <%= if function_exported?(Routes, :live_dashboard_path, 2) do %>
-              <%= link "LiveDashboard", to: Routes.live_dashboard_path(@conn, :home), class: "btn btn-sm btn-link" %>
+              <!-- <%= link "LiveDashboard", to: Routes.live_dashboard_path(@conn, :home), class: "btn btn-sm btn-link" %>-->
```
Comment (ykostov) [inline lib/studygroup_web/templates/layout/root.html.leex:27]: Well done on just muting that item, could be useful later

**Review by ykostov — CHANGES_REQUESTED** [review]
Good job on the self-initiative, just make sure to delete line 6 on routes.ex

**Review by rmilushev — APPROVED** [review]

**Review by ykostov — APPROVED** [review]
Ready to Merge

**Review by ykostov — APPROVED** [review]

**Review by ykostov — APPROVED** [review]
Good to go

**Pull request merged**